recipes/mfast/all/test_package/MDRefreshSample.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0"?>
```

Comment:
I think you can remove it now.

Reply:
@uilianries , I updated the test package. Would you mind reviewing it another time?

I really simplified it now, but kept the xml (although reduced to a couple of tags) to verify that the CMake macro was successfully exported via the `cpp_info.build_modules` mechanism.